Fix:

Loop.cpp:4224:34: error: invalid operands to binary expression ('__bind<int &, sockaddr *&, unsigned int &>' and 'int')

https://github.com/ArcticaProject/nx-libs/issues/1044
--- src/Loop.cpp.orig	2019-08-27 08:46:39.000000000 -0500
+++ src/Loop.cpp	2024-11-22 20:22:54.000000000 -0600
@@ -4221,7 +4221,7 @@
       goto SetupSocketError;
     }
 
-  if (bind(newFD, addr, addrlen) == -1)
+  if (::bind(newFD, addr, addrlen) == -1)
   {
     nxfatal << "Loop: PANIC! Call to bind failed for " << label
             << ". Error is " << EGET()
